udata/templates/reuse/display.html

```diff
@@ -87,8 +87,8 @@
                 <br/>
 
                 {# button bar #}
-                <div class="row main-toolbar">
-                    <div class="btn-toolbar">
+                <div class="row main-toolbar"><div class="col-xs-12">
```

Comment:
Yes, less indentation, less diff in this big PR ;) . I can split it on 2 lines if you prefer.